css/ampshared.css

```diff
@@ -117,6 +117,7 @@
 }
 
 .i-amphtml-layout-fill,
+.i-amphtml-layout-fill.i-amphtml-notbuilt,
```

Comment:
Please add a comment that this is specifically needed to give higher specificity to this ruleset (vs the `.i-amphtml-notbuilt` ruleset which is declared later and will win if this uses the same specificity).